test-apps/pods-embroider-app/package.json

```diff
@@ -45,17 +45,14 @@
     "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-data": "~5.3.0",
-    "ember-fetch": "^8.1.2",
     "ember-load-initializers": "^2.1.2",
     "ember-modifier": "^4.1.0",
     "ember-page-title": "^8.0.0",
     "ember-qunit": "^8.0.1",
     "ember-resolver": "^11.0.1",
     "ember-scoped-css": "workspace:^",
-    "ember-source": "~5.3.0",
+    "ember-source": "~4.10.0",
```

Comment:
I had changed this to the same version as the other embroider app in hopes the wire format would be the same as what was expected during initial implementation.

I think I may be getting trolled by a floating dependency, but I'm not sure.

It's all going to go away soon anyway.